doc/guide/listen.xml

```diff
@@ -39,6 +39,19 @@ ListenStream=
 ListenStream=443
 </programlisting>
 
+    <para>NOTE: The first empty line is intentional. <code>systemd</code> allows multiple Listen directives to be declared in a single socket unit. To change the activation port instead of adding a second port, use a full override unit instead of a snippet.</para>
```

Comment:
For consistency, I would recommend adding replacing `Listen` with `<code>Listen</code>` and inserting line breaks.